src/FormLabel/index.js

```diff
@@ -16,17 +21,53 @@ StyledAsterisk.defaultProps = {
   theme: defaultTheme,
 };
 
-const FormLabel = styled(({ className, children, required, filled, dataTest, id }) => (
-  <span className={className} data-test={dataTest} id={id}>
-    {required && (
-      <StyledAsterisk aria-hidden="true" filled={filled}>
-        *{" "}
-      </StyledAsterisk>
-    )}
-    <span>{children}</span>
-  </span>
-))`
-  display: block;
+const StyledInputErrorIcWrapper = styled.span`
```

Comment:
The icon is not properly aligned. Rebase to master and setup `align-items` properly 🙏